src/SFA.DAS.EmployerFinance/Commands/RefreshEmployerLevyData/LevyImportCleanupStrategy.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerFinance.Data;
+using SFA.DAS.EmployerFinance.Extensions;
+using SFA.DAS.EmployerFinance.Models.Levy;
+using SFA.DAS.EmployerFinance.Services;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EmployerFinance.Commands.RefreshEmployerLevyData
+{
+    public class LevyImportCleanerStrategy : ILevyImportCleanerStrategy
```

Comment:
I'm not sure this is actually an implementation of the strategy pattern.  I can't imagine there will ever be more than one?

Reply:
Agreed it's definitely a small family, but the strategy pattern only describes injecting behaviour from a family of behaviours but does not specify a minimum size of the family.